Documentation/edgefs-vdev-management.md

```diff
@@ -0,0 +1,257 @@
+Mdoffload cache | - | mdcache-TT_CHUNK_MANIFEST<br/>mdcache-TT_VERSION_MANIFEST | SSD<br/>SSD
+
+So now you are informed enough to understand next paragraphs.
+
+## Cluster Health Verification
+
+Login to the toolbox as shown in this example:
+
+```
+kubectl exec -it -n rook-edgefs 'kubectl get po --all-namespaces | awk '{print($2)}' |grep edgefs-mgr' -- env COLUMNS=$COLUMNS LINES=$LINES TERM=linux toolbox
+```
+
+To find out what device needs to go into the maintenance state, run the following command:
+
+```
+# efscli system status -v1
+ServerID CFAE1E62652370A93769378B2C862F23 ubuntu1632243:rook-edgefs-target-0 DEGRADED
+VDEVID D76242575CAA2662862CEEBC65D0B69F ata-ST1000NX0423_W470M48T ONLINE
+VDEVID B2BB69729BC3EB9ECE5F0DCB3DB4D0D6 ata-ST1000NX0423_W470NQ7A FAULTED
+VDEVID BB287F0C6747B1E59A85872B2C7F39B3 ata-ST1000NX0423_W470P9XJ ONLINE
+VDEVID 82F9A02F0D2A1BC44E6AF8D2B455189D ata-ST1000NX0423_W470M3JK ONLINE
+ServerID 1A0B10BCF8CB0E6D34451B4D3F84CE97 ubuntu1632240:rook-edgefs-target-1 ONLINE
+VDEVID 904ED942BD62FF4A997C4A23E2B8043B ata-ST1000NX0423_W470NLFR ONLINE
+...
+```
+
+From this command you will see that pod 'rook-edgefs-target-0' is in degraded state and device 'ata-ST1000NX0423_W470NQ7A' needs maintenance.
+
+## VDEV Verification
+
+Login to the affected target pod as shown in this example:
+
+```
+kubectl exec -it -n rook-edgefs rook-edgefs-target-0 -- env COLUMNS=$COLUMNS LINES=$LINES TERM=linux toolbox
+```
+
+Make sure the disk is faulted:
+
+```
+# efscli device list
+
+NAME                        | VDEV ID                          | PATH     | STATUS
++---------------------------+----------------------------------+----------+--------+
+ata-ST1000NX0423_W470M3JK   | 82F9A02F0D2A1BC44E6AF8D2B455189D | /dev/sdb | ONLINE
+ata-ST1000NX0423_W470M48T   | D76242575CAA2662862CEEBC65D0B69F | /dev/sdc | ONLINE
+ata-ST1000NX0423_W470P9XJ   | BB287F0C6747B1E59A85872B2C7F39B3 | /dev/sdd | ONLINE
+ata-ST1000NX0423_W470NQ7A   | B2BB69729BC3EB9ECE5F0DCB3DB4D0D6 | /dev/sde | UNAVAILABLE
+```
+
+Now your verification procedure can be initiated. The tool is accessible via `efscli device check` command. It provides an interactive user interface for device validation and recovery. Before getting started, a user needs to define a scratch area location. It will be used as a temporary store for environments being compacted or recovered. The scratch area can be defined in terms of data path within filesystem, a path to a raw disk (or its partition) in the /dev/ folder or raw disk/partition ID. User has to make sure the filesystem can provide enough free space to keep data from single _plevel_. The same requirement is for raw disk/partition size: at least 600GB, 1TB is recommended. A user can define the scratch area path in $(NEDGE_HOME)/etc/ccow/rt-rd.json as follow:
+
+```json
+{
+"devices": [...],
+"scratch": "/tmp.db"
+}
+```
+
+Alternatively, the path can be specified by '-s <path>' flag.<br\>
```

Comment:
This `<br\>` is visible as "code" in the rendered version

Reply:
Resolved and re-pushed.